platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/test/java/com/swirlds/platform/test/sync/SyncTests.java

```diff
@@ -1055,4 +1071,121 @@ void noEventsStartGeneration() throws Exception {
         SyncValidator.assertOnlyRequiredEventsTransferred(executor.getCaller(), executor.getListener());
         SyncValidator.assertStreamsEmpty(executor.getCaller(), executor.getListener());
     }
+
+    @Test
+    void filterLikelyDuplicatesTest() {
```

Comment:
this is not really a sync test, so probably it should be somewhere else. A sync test with filtered events would be useful I think

Reply:
Moved test to another class.

The sync test framework used in this class is super confusing to me and has very sparse javadocs. I'm not sure how I could cleanly extend it to cover this new behavior.